homeassistant/components/mobile_app/webhook.py

```diff
@@ -697,11 +700,26 @@ async def webhook_update_sensor_states(
         entry = entity_registry.async_get(entity_id)
 
         if entry and entry.disabled_by:
+            # Inform the app that the entity is disabled
             resp[unique_id]["is_disabled"] = True
 
     return webhook_response(resp, registration=config_entry.data)
 
 
+def _async_update_sensor_entity(
+    hass: HomeAssistant, entity_type: str, unique_store_key: str, data: dict[str, Any]
+) -> None:
+    """Update a sensor entity with new data."""
+    # Replace existing pending update with the latest sensor data.
+    hass.data[DOMAIN][entity_type][DATA_PENDING_UPDATES][unique_store_key] = data
```

Comment:
Is there a reason we store the data in this key order with the sensor type at the top and then the constant pending updates key below? Why not invert that? Are we going to store more things per sensor type in in hass.data?